fake-news-detection/src/fake_news_detection/pipelines/data_science/pipeline.py

```
@@ -33,4 +33,4 @@ def create_pipeline(**kwargs) -> Pipeline:
             outputs=["predictions", "evaluation_report"],
             name="train_and_evaluate_node",
         ),
-    ])
+    
```

Ruff (0.11.9)

Blank line contains whitespace

(W293)